### PR TITLE
[BUD-80] Typography

### PR DESCRIPTION
**Pull request by lucianojss**
# Typography

# [#BUD-80](https://digitalrig.atlassian.net/browse/BUD-80) 

## Description

Gravity fonts added.

## Additional comments

I wanted to do it in a more fancy way with `webfontloader` etc.. but I couldn't manage so I went with the most traditional way with `.css` file and webpack.
Let me know your thoughts.

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+
 import App from './components/App';
 import * as serviceWorker from './serviceWorker';
+import './index.css';
```
Comment (michaelmoney) [inline src/index.tsx]: I've question here:  Is this the only way to define _global_ font for the app by using `index.css` in `index.ts` file? CSS is not flexible format. Have you tried this (to load fonts using JS and expose them in the `theme.ts`)?
image
From here: https://material-ui.com/customization/typography/

Comment (michaelmoney) [reply src/index.tsx]: Oh sorry @lucianojss. I've read your description. 😀
>I wanted to do it in a more fancy way with webfontloader etc.. but I couldn't manage so I went with the most traditional way with .css file and webpack.
Let me know your thoughts.

If so, maybe I'm gonna look at it later. Let's keep it as it is.
Approved :white_check_mark:

File: src/styles/theme.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ let theme = createMuiTheme({
     button: {
       textTransform: 'none',
     },
+    fontFamily: 'Wipro Akkurat TT, Wipro Akkurat Mono TT, Roboto, Helvetica, Arial, sans-serif',
```
Comment (duiliopastorelli) [inline src/styles/theme.ts]: Is Roboto imported somewhere in the App? I don't think it's natively supported by all browsers except Chrome.
If I'm right it's a weak fallback.

Comment (lucianojss) [reply src/styles/theme.ts]: That's why it ends up with `Helvetica, Arial, sans-serif` :)

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,62 @@
+@font-face {
```
[the remark this answers is not in the document]
Comment (lucianojss) [reply src/index.css]: Only the fonts that are required in the page are loaded

[the remark this answers is not in the document]
Comment (duiliopastorelli) [reply src/index.css]: This has been discussed, for now I think it's fine to have them here as they don't impact performances, but still I am slightly concerned of giving so many fonts available for further development in a simple app.
I hope we will find a way to import them through WebPack though in the future.

File: src/index.tsx (hunk 1 above)
Comment (lucianojss) [reply src/index.tsx]: Sure @duiliopastorelli

**Pull request merged**